templates/planka.xml

```diff
@@ -1,6 +1,7 @@
 <?xml version="1.0"?>
 <Container version="2">
   <Name>planka</Name>
+  <Icon>templates\img\planka.png</Icon>
```

Comment:
Icon needs to be the path to the raw GitHub content, see https://github.com/selfhosters/unRAID-CA-templates/blob/c902bcf838a274c482d9f905d86fb87890849bfb/templates/adminer.xml#L19